docs/src/man/comparisons.md

```diff
@@ -1,77 +1,173 @@
+indicate the last index.
+
+### Common operations
+
+| Operation                | pandas                                                | DataFrames.jl                                                 |
+|:-------------------------|:------------------------------------------------------|:--------------------------------------------------------------|
+| Reduce multiple values   | `df['z'].mean(skipna = False)`                        | `combine(df, :z => mean)`                                     |
+|                          | `df['z'].mean()`                                      | `combine(df, :z => mean ∘ skipmissing => :z_mean)`            |
+| Add new columns          | `df.assign(x_mean = df['x'].mean())`                  | `transform(df, :x => mean => :x_mean)`                        |
+| Rename columns           | `df.rename(columns = {'x': 'x_new'})`                 | `rename(df, :x => :x_new)`                                    |
+| Pick & transform columns | `df.assign(x_mean = df['x'].mean())[['x_mean', 'y']]` | `select(df, :x => mean, :y)`                                  |
+| Sort rows                | `df.sort_values(by = ['x'])`                          | `sort(df, :x)`                                                |
+
+Note that Julia propagates `missing` data by default for safety reasons. The `skipmissing` function
+can be used to remove missing data. See more details at the [Additional Differences](@ref) section below.
+
+### Grouping data and aggregation
+
+| Operation                    | pandas                                                                                    | DataFrames.jl                               |
+|:-----------------------------|:------------------------------------------------------------------------------------------|:--------------------------------------------|
+| Aggregate by groups          | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean)`    |
+| Aggregate and add columns    | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')`                       | `transform(groupby(df, :grp), :x => mean)`  |
+| Aggregate and select columns | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')[['grp','x_mean','y']]` | `select(groupby(df, :grp), :x => mean, :y)` |
+
+### More advanced commands
+
+| Operation                 | pandas                                                                       | DataFrames.jl                                                 |
+|:--------------------------|:-----------------------------------------------------------------------------|:--------------------------------------------------------------|
+| Complex Function          | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => (v -> mean(cos, skipmissing(v))))`         |
```

Comment:
same here. Also one pair of `(` and `)` is not needed in Julia code

Reply:
Fixed, thanks.